package.json

```diff
@@ -7,11 +7,10 @@
     "url": "https://github.com/withastro/compiler.git"
   },
   "scripts": {
-    "build": "make wasm && lerna run build --scope @astrojs/compiler",
-    "build:compiler": "lerna run build --scope @astrojs/compiler",
+    "build": "make wasm",
```

Comment:
```suggestion
    "build": "make wasm",
    "build:compiler": "pnpm --filter @astrojs/compiler run build",
```

Reply:
@natemoo-re You know, I looked for like an hour for that `--filter` command. LOL